services/credentials-service/src/credentials/credentials.service.ts

```diff
@@ -319,4 +319,44 @@ export class CredentialsService {
       return { id: cred.id, ...signed };
     });
   }
+
+  async getRevocationList(
+    issuerId: string,
+    page = 1,
+    limit= 1000,
+  ){
+    let revocationList: any;
+
+    if (issuerId === "") {
+      throw new InternalServerErrorException('Please provide a valid issuer ID');
+    }
+    
+    try {
+      revocationList = await this.prisma.verifiableCredentials.findMany({
+        where: {
+          issuer: issuerId,
+          status: VCStatus.REVOKED,
+        },
+        select: {
+          id: true,
+          status: true,
+          tags : true,
+          issuer : true,
+          issuanceDate: true,
+          expirationDate: true,
+          credential_schema : true,
+        },
+        skip: (page -1) * limit,
+        take: limit,
+        orderBy: {
+          issuanceDate: 'desc',
+        },
+      }); 
+    } catch (error) {
+      this.logger.error('Error fetching RevocationList');
+      throw new InternalServerErrorException('Error fetching revocationList');
```

Comment:
Which HTTP status code is triggered when an exception occurs.

Reply:
This function throws a 500 status code.